its/plugin/tests/src/test/java/com/sonar/it/java/suite/JavaTestSuite.java

```diff
@@ -58,20 +59,24 @@
 public class JavaTestSuite {
 
   public static final FileLocation JAVA_PLUGIN_LOCATION = FileLocation.byWildcardMavenFilename(new File("../../../sonar-java-plugin/target"), "sonar-java-plugin-*.jar");
+  public static final FileLocation TUTORIAL_EXAMPLE_PLUGIN_LOCATION = FileLocation.byWildcardMavenFilename(new File("../../../docs/java-custom-rules-example/target"), "java-custom-rules-example-*.jar");
 
   @ClassRule
   public static final Orchestrator ORCHESTRATOR;
 
   static {
     OrchestratorBuilder orchestratorBuilder = Orchestrator.builderEnv()
-      .setSonarVersion(System.getProperty("sonar.runtimeVersion", "LATEST_RELEASE[7.9]"))
+      .setSonarVersion(System.getProperty("sonar.runtimeVersion", "LATEST_RELEASE[8.7]"))
```

Comment:
Should we go for `LATEST_RELEASE` to make sure we are always testing with the latest?

Reply:
shit I should not have committed this